www/include/eventLogs/viewLog.php

```diff
@@ -130,20 +158,22 @@
 }
 
 $defaultServicegroups = array();
-if (isset($_GET['svcg'])) {
-    $svcg = explode(",", $_GET['svcg']);
+if (isset($getInputs['svcg'])) {
+    $svcg = explode(",", $getInputs['svcg']);
+    $svcGroupsIds = filteredArrayId($svcg);
     $serviceGrpObj = new CentreonServicegroups($pearDB);
-    $serviceGrpArray = $serviceGrpObj->getServicesGroups($svcg);
+    $serviceGrpArray = $serviceGrpObj->getServicesGroups($svcGroupsIds);
     foreach ($serviceGrpArray as $defaultServicegroup) {
         $defaultServicegroups[$defaultServicegroup['name']] = $defaultServicegroup['id'];
     }
 }
 
 $defaultPollers = array();
-if (isset($_GET['poller'])) {
-    $poller = explode(",", $_GET['poller']);
+if (isset($getInputs['poller'])) {
+    $poller = explode(",", $getInputs['poller']);
+    $pollerIds = filteredArrayId($pollerIds);
     $pollerObj = new CentreonInstance($pearDB, $pearDBO);
-    $pollerArray = $pollerObj->getInstancesMonitoring($poller);
+    $pollerArray = $pollerObj->getInstancesMonitoring($pollerIds);
```

Comment:
The verification of ids should not be done here but directly in the getInstancesMonitoring method to get a wider scope of application.